src/test/java/SetConfigTest.java

```diff
@@ -94,4 +102,39 @@ void testPlayerSetTeams() {
         Assertions.assertTrue(scoreboard.getEntityTeam(stefan).getName().equals("Red"));
         Assertions.assertTrue(scoreboard.getEntityTeam(jawad).getName().equals("Yellow"));
     }
+
+    @Test
+    @DisplayName("Test that the recipe for creating a golden apple from a player head works")
+    void testPlayerCanCraftGoldenAppleFromPlayerHead() {
+        PlayerMock player = server.addPlayer();
+        player.setOp(true);
+        server.execute("uhc", player, "set", "player.head.golden.apple=true");
+        ItemStack apple = new ItemStack(Material.GOLDEN_APPLE);
+        NamespacedKey key = new NamespacedKey(plugin, "golden_apple");
+        ShapedRecipe recipe = new ShapedRecipe(key, apple)
+                .shape("   ", " X ", "   ")
+                .setIngredient('X', Material.PLAYER_HEAD);
+        Bukkit.addRecipe(recipe);

```

Comment:
this line shouldn't be needed because the recipe should already have been added by the plugin, no?

Reply:
yep you right :)